util/testutil/integration/containerd.go

```diff
@@ -69,8 +64,9 @@ func (c *containerd) New(opt ...SandboxOpt) (sb Sandbox, cl func() error, err er
 	}
 
 	deferF := &multiCloser{}
-	cl = deferF.F()
+	cl := deferF.F()
 
+	var err error
```

Comment:
Fixed.

Reply:
FWIW, I don't tend to like this pattern for exactly this reason. Small changes tend to have unforeseen effects and it ends up being very brittle. That being said, I was specifically trying to correct it in this case and just didn't finish, so I should have seen them and it's not really the patterns fault here :)

Reply:
@tiborvass that's a good idea. I'm not aware of any existing check, but this might be a good fit for go lint even (new lint rules are restricted to the things defined on this eiki page, which has something at least tangentially related: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)